samples/Azure.AI.FormRecognizer/Generated/AzureAIFormRecognizerModelFactory.cs

```diff
@@ -213,26 +196,22 @@ public static DataTable DataTable(int rows = default, int columns = default, IRe
         /// <param name="isHeader"> Is the current cell a header cell?. </param>
         /// <param name="isFooter"> Is the current cell a footer cell?. </param>
         /// <returns> A new <see cref="Models.DataTableCell"/> instance for mocking. </returns>
-        public static DataTableCell DataTableCell(int rowIndex = default, int columnIndex = default, int? rowSpan = default, int? columnSpan = default, string text = default, IReadOnlyList<float> boundingBox = default, float confidence = default, IReadOnlyList<string> elements = default, bool? isHeader = default, bool? isFooter = default)
+        public static DataTableCell DataTableCell(int rowIndex = new int(), int columnIndex = new int(), int? rowSpan = null, int? columnSpan = null, string text = null, IEnumerable<float> boundingBox = null, float confidence = new float(), IEnumerable<string> elements = null, bool? isHeader = null, bool? isFooter = null)
```

Comment:
`int rowIndex = new int()` is one of the things where I'd prefer to have `int rowIndex = default`.

Reply:
Why not keep default everywhere?

Reply:
It is produced by `CodeWriterExtensions.WriteConstant`. I probably can change case for `NewInstanceSentinel` for value types without any problems. For reference types, it may not be safe to do in a general method like `WriteConstant`.

Reply:
Or I can change it in `WriteParameterNullChecks`. Let me try.

Reply:
>For reference types, it may not be safe to do in a general method like WriteConstant

Why not, I don't remember the difference between null and default.

Reply:
Difference happens when calling overloads:
```cs
static void Me(object o) => Console.WriteLine("object");
static void Me(int i) => Console.WriteLine("int");

Me(null); // prints "object"
Me(default); // prints "int"
```

Reply:
But not for parameters, right?

Reply:
No, for parameters it's ok. But if I change it for both value types and reference types, PR will become huge. So I've changed it for value types only.